index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I can see you did great work 👍, but you need to fix some points.

- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎. 
- [ ] Kindly, make sure to highlight the effort of the student 👍   
- [ ]  Please consider always including an explanation, a "why", for each requested change. This means to describe the user why he is being asked for some change.

## General example:-

This is a well-explained request: `Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible`

This request lacks a "why":` Why don't you use an article tag here instead of a div?`